packages/coze-uniapp/src/ws-tool/pcm-stream-player/index.ts

```
@@ -0,0 +1,617 @@
+      ) {
+        // Convert from 16-bit PCM to float [-1.0, 1.0] on-the-fly
+        outputBuffer[i] = this.currentBuffer[this.playbackPosition++] / 0x8000;
+      } else {
+        // Current buffer is exhausted, try to get next buffer
+        if (!this.getNextBuffer()) {
+          // No more data, fill rest with silence
+          outputBuffer[i] = 0;
+          this.isProcessing = i === outputBuffer.length - 1 ? false : true;
+        } else {
+          // Got new buffer, use its first sample
+          outputBuffer[i] = this.currentBuffer
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)